chapter_appendix/information_theory.md

```diff
@@ -0,0 +1,371 @@
+# Information Theory
+:label:`chapter_information_theory`
+
+Machine learning, which studies how computer systems can use data to improve performance, is aimed to extract critical and interesting signals from data. While, information theory, which studies encoding, decoding, transmitting and manipulating information (aka data), can help us measure and compare how much information are presented in different signals. As a result, information theory provides a strong backbone to the fundamental of machine learning. For example, “cross entropy loss” is used as an objective function in many machine learning algorithms. This is a direct application of an information theory to measure the loss. In addition, in the rule-based learning algorithm (such as the decision tree algorithm), we can use entropy as a unit to quantify the amount of information gained from the best split at each node. In this section, we will talk about the most fundamental concept of information theory and its applications via several examples.
+
```

Comment:
Such a loss is a direct application to information theory.

To name a few, on the one hand,...; on the other hand,...
Or
Here are two examples.

Reply:
This section, by several examples, we will illustrate the fundamental concepts of information theory and their application.